src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetupAction.java

```diff
@@ -95,6 +98,37 @@ public void actionPerformed(@NotNull AnActionEvent e) {
                     // Show Pricing alert dialog.
                     PricingAlerts.setPlanLimitReached(project);
                 } else {
+                    // Get access token
```

Comment:
Why do we have this block of code? We don't need any separate API call except the existing REPO-INIT call, right?